node/chainSimulator/testOnlyProcessingNode_test.go

```diff
@@ -30,8 +31,12 @@ func createMockArgsTestOnlyProcessingNode(t *testing.T) ArgsTestOnlyProcessingNo
 	err = LoadConfigFromFile(pathForPrefsConfig, &prefsConfig)
 	assert.Nil(t, err)
 
+	workingDir, err := os.Getwd()
```

Comment:
we can remove this and use t.TempDir() instead

Reply:
done